CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ x.x.x
 * Dropped support for Python 2.7 and 3.4-3.5.
 * Added Python 3.8 and 3.9 testing.
 * Upgraded Flask to 2.0.3.
+* Update the list of contributors
```

Comment:
Personally, I wouldn't include this in the CHANGELOG.md, but if you think we should, I would change "Update" to "Updated" for consistency. Also for consistency, I would add periods at the end of the added lines.

Reply:
I've removed it. I wasn't sure about adding it in the first place :smile: